CHANGELOG.txt

```diff
@@ -1,3 +1,7 @@
+* Added default headers to server response, so that if a response from
+  backend didn't contain any server header it will automatically add a
+  daphne server header.
```

Comment:
The header was already there for websockets, so I think we could reword this to:

```suggestion
* Always add the `Server` header, with value `Daphne`, when it isn’t already
  present.
```

What do you think?

Reply:
Seems good to me. Thank you for suggesting this 🥳

Reply:
Sure your wording is good.

Reply:
Sure, perhaps:

```
if the app doesn't set a `Server` header.
```

There isn't really a separate "backend" from Daphne - it's your "backend server" already.

Reply:
Updated 😄